src/services/administration/iam/policy/lib/config.ts

```diff
@@ -1,5 +1,17 @@
 import { KeyItem } from '@spaceone/console-core-lib/component-util/query-search/type';
 
+export const POLICY_TYPES = Object.freeze({
+    MANAGED: 'MANAGED',
+    CUSTOM: 'CUSTOM',
+} as const);
+export type POLICY_TYPES = typeof POLICY_TYPES[keyof typeof POLICY_TYPES];
+
+export const POLICY_STATE = {
+    ENABLED: 'ENABLED',
+    DISABLED: 'DISABLED',
+} as const;
+export type POLICY_STATE = typeof POLICY_STATE[keyof typeof POLICY_STATE];
```

Comment:
```suggestion
export type PolicyState= typeof POLICY_STATE[keyof typeof POLICY_STATE];
```

타입명 컨벤션 욜케 가져가기로 합의되었슴다